lib/block-supports/behaviors.php

```diff
@@ -44,6 +44,15 @@ function gutenberg_register_behaviors_support( $block_type ) {
  * @return string                Filtered block content.
  */
 function gutenberg_render_behaviors_support_lightbox( $block_content, $block ) {
+
+	// We've deprecated the lightbox implementation via behaviors.
+	// While we may continue to explore behaviors in the future, the lightbox
+	// logic seems very specific to the image and will likely never be a part
+	// of behaviors, even in the future. With that in mind, we've rewritten the lightbox
+	// to be a feature of the image block and will also soon remove the block_supports.
+	// See https://github.com/WordPress/gutenberg/issues/53403.
+	_deprecated_function( 'gutenberg_render_behaviors_support_lightbox', '17', '' );
```

Comment:
I've added the `_deprecated_function()` call here. I was considering how to prevent duplication of this logic between here and the `index.php`, but couldn't think of a good way to do it, so I've left it intact considering it will be removed soon anyway.

Reply:
Yeah, I think that's fine. So long as it's documented in the comments.